[UPDATE] Add a link to linear scaling on redis.com (#2151)
redis/redis-doc: docs/manual/scaling.md

  - /topics/partitioning
---

Redis scales horizontally with a deployment topology called Redis Cluster. 
This topic will teach you how to set up, test, and operate Redis Cluster in production.
You will learn about the availability and consistency characteristics of Redis Cluster from the end user's point of view.

If you plan to run a production Redis Cluster deployment or want to understand better how Redis Cluster works internally, consult the [Redis Cluster specification](/topics/cluster-spec). To learn how Redis Enterprise handles scaling, see [Linear Scaling with Redis Enterprise](https://redis.com/redis-enterprise/technology/linear-scaling-redis-enterprise/).

## Redis Cluster 101

Redis Cluster provides a way to run a Redis installation where data is automatically sharded across multiple Redis nodes. 

> Redis Cluster also provides some degree of availability during partitions&mdash;in practical terms, the ability to continue the operations when some nodes fail or are unable to communicate. 

Reviewer: ```suggestion
Redis Cluster also provides some degree of availability during partitions&mdash;in practical terms, the ability to continue operations when some nodes fail or are unable to communicate. 
```

> However, the cluster stops to operate in the event of larger failures, for example, when the majority of masters are unavailable.

Reviewer: ```suggestion
However, the cluster will become unavailable in the event of larger failures (for example, when the majority of masters are unavailable).
```
